elements/lisk-cryptography/test/bls.spec.ts

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable camelcase */
```

Comment:
Is this needed?

Reply:
Yes the eth specs files contains keys which are not camelcase. So have to disable it for all the file.